runtime/runtime/src/balance_checker.rs

```diff
@@ -41,7 +42,7 @@ fn receipt_cost(
     Ok(match &receipt.receipt {
         ReceiptEnum::Action(action_receipt) => {
             let mut total_cost = total_deposit(&action_receipt.actions)?;
-            if !AccountId::is_system(&receipt.predecessor_id) {
+            if !AccountIdRef::is_system(&receipt.predecessor_id) {
```

Comment:
I feel this should work (and then AccountIdRef does not need to be imported explicitly):

```suggestion
            if !receipt.predecessor_id.is_system() {
```

Reply:
@ruseinov I see you updated it in one place, but there are a few other places there to be changed. Please, update those as well.